### ISSUE TITLE
I can't stop logging health check GET /healthz on ESPv2 Pod

### ISSUE THREAD

**Issue by alejorod18** [GoogleCloudPlatform/esp-v2#749]
I have multiple ESPv2 services in my Kubernetes cluster. A those services generate logs by each kubernetes readiness or liveness prove and for each google global load balancer liveness prove.
![espv2 drawio](https://user-images.githubusercontent.com/20443183/200035712-c1213621-4613-42aa-8138-32d5aa8d6081.png)

Because I have multiple of these solutions running in the same cluster, the cluster generate a lot of GET /healthz logs. 
<img width="1179" alt="image" src="https://user-images.githubusercontent.com/20443183/200037165-6ae0f53d-3c35-4ab0-a563-f6390795274f.png">
[downloaded-logs-20221104-122130.csv](https://github.com/GoogleCloudPlatform/esp-v2/files/9940709/downloaded-logs-20221104-122130.csv)

In conclusion, my problem is that I can't bypass the GET /healthz logs, and those logs don't add value to my troubleshooting, otherwise those logs create unnecessary cost and unnecessary cognitive load.

Please I ask for your help to reduce my logs costs, by skipping those logs in the ESPv2 container.

thanks.

**Comment by qiwzhang** [comment]
It looks like these logs are from GCLB,   could you open it up, and get the details of a log entry?
